.github/workflows/ci.yml

```diff
@@ -15,15 +15,15 @@ env:
 
 jobs:
   unit-tests:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     strategy:
       fail-fast: false
 
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
         with:
-          python-version: '3.6'
+          python-version: '3.6.15'
```

Comment:
Checked out the [python-versions](https://github.com/actions/python-versions) in github action. It seems that the lowest supported version for 3.6 is 3.6.7. We probably should use the lowest supported version here.